lib/matplotlib/animation.py

```diff
@@ -1331,6 +1331,12 @@ def to_jshtml(self, fps=None, embed_frames=True, default_mode=None):
             What to do when the animation ends. Must be one of ``{'loop',
             'once', 'reflect'}``. Defaults to ``'loop'`` if the *repeat*
             parameter is True, otherwise ``'once'``.
+
+        Returns
+        -------
+        str
+            An HTML representation of the animation embedded as a js object as
+            produced with the *HTMLWriter*.
```

Comment:
```suggestion
            produced with the `.HTMLWriter`.
```
To link to the docs.